plugins/catalog/api-report.md

```diff
@@ -138,6 +138,8 @@ export interface CatalogTableProps {
   actions?: TableProps<CatalogTableRow>['actions'];
   // (undocumented)
   columns?: TableColumn<CatalogTableRow>[];
+  // (undocumented)
+  options?: TableProps<CatalogTableRow>['options'];
```

Comment:
think `tableOptions` is probably a better name here.

Reply:
Updated. changed to tableOptions